styles/base.css

```diff
@@ -48,6 +48,20 @@ body {
     }
 }
 
+:root {
+    --main-container-first-row-margin: 104px 0;
+}
+
+@media (min-width: 992px) {
+    :root {
+        --main-container-first-row-margin: 114px 0;
+    }
+}
+
+main.container-fluid > .row:first-child {
+    margin: var(--main-container-first-row-margin);
+}
+
```

Comment:
This will be removed once #135 is merged. I wanted to test that these styles work well with the blog.